package.json

```diff
@@ -22,6 +22,7 @@
     "@octokit/plugin-throttling": "^3.5.2",
     "@octokit/rest": "^18.12.0",
     "common-tags": "^1.8.2",
+    "detect-package-manager": "^3.0.1",
```

Comment:
Shouldn't there be a yarn.lock update that comes with this :D?

Reply:
We'll also need the dist update. I think I'll just pick up where you left this :)